force-app/main/default/classes/NextStepQueueable.cls

```diff
@@ -17,42 +17,57 @@
  * 
  * Note: The class below contains placeholders for implementing a queueable job to update accounts and opportunities.
  */
-public with sharing class NextStepQueueable {
+public with sharing class NextStepQueueable implements Queueable{
 
     // Member variable to store the accounts
-    private Map<Id,Account> accounts;
+    private Map<Id,Account> accounts = new Map<Id,Account>();
 
     // Constructor to accept the accounts
     public NextStepQueueable(Map<Id,Account> accounts) {
         // Store the accounts in a member variable
+        this.accounts.putAll(accounts);
     }
 
     // Method to queue the job
     public static void enqueueJob() {
+        Map<Id, Account> accountsToUpdate = new Map<Id, Account>();
         // Get 10 accounts with Next Step populated
+        for(Account acc : [SELECT Id FROM Account WHERE Next_Step__c != null LIMIT 10]){
+            accountsToUpdate.put(acc.Id, acc);
+        }
```

Comment:
Remember the shorthand for creating a Map<Id, sObject> from a query - https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_map_sobject.htm

Reply:
Thanks Tony. This will come in handy.